extensions/PhabBugz/lib/Feed.pm

```diff
@@ -255,11 +258,10 @@ sub group_query {
             );
         }
 
-        if ( my @group_members = get_group_members($group) ) {
-            INFO("Setting group members for " . $project->name);
-            $project->set_members( \@group_members );
-            $project->update();
-        }
+        INFO("Setting group members for " . $project->name);
+        my @group_members = get_group_members($group);
+        $project->set_members( \@group_members );
+        $project->update();
```

Comment:
i assume this is a no-op when `@group_members` is empty

Reply:
Actually not really. If there are no members of a group in BMO, then there will be no members in the related project in Phabricator. This is by design.